app/UncaughtErrorHandler.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\UiTPASService;
+
+use Sentry\State\HubInterface;
+use Throwable;
+
+class UncaughtErrorHandler
+{
+    /** @var HubInterface */
+    private $sentryHub;
+
+    public function __construct(HubInterface $sentryHub)
+    {
+        $this->sentryHub = $sentryHub;
+    }
+
+    public function handle(Throwable $throwable): void
+    {
+        if ($throwable->getCode() !== 404) {
+            $this->sentryHub->captureException($throwable);
+        }
```

Comment:
Maybe to clarify for @jdrieghe since we discussed this while pairing over Tuple:
This app is a small internal API, to bridge the Angular app (and later the Vue app) and the XML UiTPAS API. Since we can sometimes get errors from that API and probably want a signal that something is wrong downstream, we decided to track everything but 404 errors in Sentry, even if it was already handled before and returning an API Problem JSON response.